test/box-luatest/gh_9679_netbox_on_schema_reload_error_behaviour_test.lua

```diff
@@ -0,0 +1,28 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+-- Tests the behaviour of the `on_schema_reload` trigger.
+g.test_on_schema_reload_error_behaviour = function(cg)
+    local trigger_err = '777'
+    cg.server:exec(function(trigger_err)
+        local net_box = require('net.box')
+
+        local c = net_box.connect(box.cfg.listen, {wait_connected = false})
+        c:on_schema_reload(function()
+            error(trigger_err)
+        end)
+        t.assert(c:wait_state('active', 0.1))
```

Comment:
Let's not try to save on timeouts :)

A small timeout is a 100% recipe for the test to be flaky in CI. Try to run the test like this 
`./test-run.py $(yes 9679 | head -n 512)` and you will see some errors:
```
[013] box-luatest/gh_9679_netbox_on_schema_reload_er>                 [ fail ]
[013] Test failed! Output from reject file /tmp/t/rejects/box-luatest/gh_9679_netbox_on_schema_reload_error_behaviour.reject:
[013] Tarantool version is 3.1.0-entrypoint-162-g4170c5012d
[013] TAP version 13
[013] 1..1
[013] # Started on Fri Mar 22 13:08:17 2024
[013] # Starting group: box-luatest.gh_9679_netbox_on_schema_reload_error_behaviour
[013] not ok 1	box-luatest.gh_9679_netbox_on_schema_reload_error_behaviour.test_on_schema_reload_error_behaviour
[013] #   ...gh_9679_netbox_on_schema_reload_error_behaviour_test.lua:25: expected: a value evaluating to true, actual: false
[013] #   stack traceback:
[013] #   	...gh_9679_netbox_on_schema_reload_error_behaviour_test.lua:18: in function 'box-luatest.gh_9679_netbox_on_schema_reload_error_behaviour.test_on_schema_reload_error_behaviour'
[013] #   	...
[013] #   	[C]: in function 'xpcall'
[013] #   artifacts:
[013] #   	server -> /private/tmp/t/013_box-luatest/artifacts/server-60e8GuD-jf_e
[013] # Ran 1 tests in 1.664 seconds, 0 succeeded, 1 failed
[013] 
```
I think increasing it to a second would work just fine. Besides, this doesn't make the test run longer.

Reply:
Sorry, haven't thought about this previously. Changed the timeout to 10 seconds.